src/vconsole/vconsole-setup.c

```diff
@@ -470,6 +471,8 @@ int main(int argc, char **argv) {
         if (r < 0 && r != -ENOENT)
                 log_warning_errno(r, "Failed to read /proc/cmdline, ignoring: %m");
 
+        vc_keymap = !isempty(vc_keymap_alloc) ? vc_keymap_alloc : SYSTEMD_DEFAULT_KEYMAP;
```

Comment:
drop the ! and switch the ternary params? saves one byte!

(i mean, seriously, i find the needless negation here a bit confusing)